apps/yaft/fb/common.cpp

```diff
@@ -189,18 +196,18 @@ draw_line(rmlib::fb::FrameBuffer& fb, struct terminal_t* term, int line) {
 
   /* actual display update (bit blit) */
   // TODO: group updates.
-  fb.doUpdate(
-    { { 0, y_start }, { fb.canvas.width() - 1, y_start + CELL_HEIGHT - 1 } },
-    rmlib::fb::Waveform::DU,
-    rmlib::fb::UpdateFlags::None);
-  update_count++;
-
-  /* TODO: page flip
-          if fb_fix_screeninfo.ypanstep > 0, we can use hardware panning.
-          set fb_fix_screeninfo.{yres_virtual,yoffset} and call
-     ioctl(FBIOPAN_DISPLAY) but drivers  of recent hardware (inteldrmfb,
-     nouveaufb, radeonfb) don't support... (maybe we can use this by using
-     libdrm) */
+  if (term->landscape) {
+    fb.doUpdate(
+      { { y_start, 0 }, { y_start - CELL_HEIGHT, term->width - 1 } },
```

Comment:
Maybe use a ternary for just the rect parameter? That way there's a single location that defines the waveform and update flags.